source/index.rst

```diff
@@ -1,7 +1,10 @@
 Welcome to the COCO Documentation
 =================================
 
-Learn all about COCO here. We introduce you to the concepts and architecture, allow you to get started quickly with the various platform API's with explained code examples, and finally allow you to deep dive into the API Documentation and build your own COCO-based solution.
+Learn all about COCO here. We introduce you to the concepts and architecture,
```

Comment:
I don't see any change in the content. Please revert this file to my version. The line breaks here don't make any difference to how the file appears in HTML or the RST preview.